### PR TITLE
Epiphane branch

### PR DESCRIPTION
**Pull request by Whryne**
Merging finished header and footer

### CODE REVIEW

**Comment by epistar1** [comment]
good

**Review by epistar1 — APPROVED** [review]

File: footer/footer.css [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+.sponsors{
+  display: grid;
+  grid-template-rows: auto auto;
+  grid-template-columns: auto auto;
+
+}
+
+span{
```
Comment (SimsCAustin) [inline footer/footer.css:225]: Is this needed?

Comment (SimsCAustin) [reply footer/footer.css:225]: The span tag in css

**Review by SimsCAustin — COMMENTED** [review]
Is the span tag in the css needed? It appears to be empty.

**Review by colemancan007 — APPROVED** [review]

**Comment by epistar1** [comment]
pull request done for navbar and footer.

**Pull request merged**